products/views.py

```diff
@@ -2,24 +2,63 @@
 
 from django.views         import View
 from django.http          import JsonResponse
+from django.db.models     import Q
 
 from products.models      import *
 
 class CategoryList(View):
     def get(self,request):
         categories = Category.objects.all()
-
+        
         result = [{
             'id'            : category.id,
             'category_name' : category.name,
             'subcategory_list' :[{
                 'id'   : subcategory.id,
                 'name' : subcategory.name,
+                'subcategory_image': subcategory.image_url,
                 'product_list' : [{
                     'id'   : product.id,
                     'name' : product.korean_name 
                         } for product in subcategory.products.all()]
                 } for subcategory in category.sub_categories.all()]
             } for category in categories]
 
-        return JsonResponse({'message':'SUCCESS' ,'Category' : result}, status=200)
\ No newline at end of file
+        return JsonResponse({'message':'SUCCESS' ,'Category' : result}, status=200)
+
+class ProductList(View):
+    def get(self,request):
+        try:
+            category_name    = request.GET.get('category',None).split(',')
+            subcategory_name = request.GET.getlist('subcategory', None)
+            sorting          = request.GET.get('sort', None)
```

Comment:
default로 None을 넣게되면 Error가 발생하지 않나요?
```suggestion
            sorting          = request.GET.get('sort', 'id')
```

Reply:
디폴트에 None 넣으니까 에러나서 제가 if sorting을 따로 뺐습니다.
수정했습니다